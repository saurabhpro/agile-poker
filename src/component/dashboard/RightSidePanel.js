import React from 'react';
import { Card, Button, Row } from 'react-bootstrap';
import Team from './team/Team';

const style = { width: '18rem', margin: '1em', border: 'none' };

export default function RightSidePanel() {
  return (
    <div>
      <Row>
        <Card style={style}>
          <Card.Body>
            <Card.Title>Team</Card.Title>

            <Team />
          </Card.Body>
        </Card>
      </Row>
      <Row>
        <Card style={style}>
          <Card.Body>
            <Card.Title>Poker</Card.Title>
            <Card.Text>Just select one</Card.Text>
            <Button variant="primary">Reset</Button>
          </Card.Body>
        </Card>
      </Row>
    </div>
  );
}
