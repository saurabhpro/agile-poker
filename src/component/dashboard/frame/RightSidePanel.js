import React from 'react';
import PropTypes from 'prop-types';
import { Container } from 'react-bootstrap';

import RightPanelActionCard from './right-panel/RightPanelActionCard';
import RightPanelTeamStatusCard from './right-panel/RightPanelTeamStatusCard';

const style = { border: 'none' };

export default function RightSidePanel({ currentUser }) {
  return (
    <div>
      {currentUser && (
        <Container className="justify-content-center">
          <RightPanelActionCard
            currentUser={currentUser}
            style={style}
          />
          <RightPanelTeamStatusCard style={style} />
        </Container>
      )}
    </div>
  );
}

RightSidePanel.propTypes = {
  currentUser: PropTypes.object.isRequired,
};
