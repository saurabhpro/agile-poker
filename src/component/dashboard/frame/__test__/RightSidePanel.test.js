import React from 'react';
import { shallow } from 'enzyme';
import RightSidePanel from '../RightSidePanel';

let component;

describe('sign in page', () => {
  beforeAll(() => {
    component = shallow(<RightSidePanel userName="Saurabh" />);
    console.debug(component.debug());
  });

  it('matches the rendered snapshot', () => {
    expect(component).toMatchSnapshot();
  });

  it('should render one <RightPannelActionCard>', () => {
    expect(component.find('RightPannelActionCard')).toHaveLength(0);
  });

  it('should render one <RightPannelTeamStatusCard>', () => {
    expect(component.find('RightPannelTeamStatusCard')).toHaveLength(
      0,
    );
  });
});
