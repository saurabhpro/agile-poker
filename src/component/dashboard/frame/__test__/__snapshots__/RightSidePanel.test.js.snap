// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sign in page matches the rendered snapshot 1`] = `ShallowWrapper {}`;
